pyspark/bigdl/util/tf_utils.py

```diff
@@ -117,9 +117,14 @@ def save_variable_bigdl(tensors, target_path, bigdl_type="float"):
     :param bigdl_type: model variable numeric type
     :return: nothing
     """
+    import numpy as np
     jtensors = {}
     for tn in tensors.keys():
-        jtensors[tn] = JTensor.from_ndarray(tensors[tn])
+        if not isinstance(tensors[tn], np.ndarray):
```

Comment:
when tensors[tn] will be an array?

Reply:
when the variable is a scalar, tensors[tn] is not a `np.ndarray` but a single number. `JTensor.from_ndarray()` will fail on the single number case.

Reply:
Same here fix single number fail np.ndarary ?